apps/minaintyg/src/components/Layout/Breadcrumbs.tsx

```diff
@@ -0,0 +1,39 @@
+import { IDSBreadcrumbs, IDSCrumb } from '@frontend/ids-react-ts'
+import { Link, Params, useMatches } from 'react-router-dom'
+
+type Match = ReturnType<typeof useMatches>[number]
+type MatchWithCrumb = Match & {
+  handle: { crumb: (data: Params<string>) => string }
+}
+
+function hasCrumb(match: MatchWithCrumb | Match): match is MatchWithCrumb {
```

Comment:
This could be moved out to utils

Reply:
Agreed, this would increase the testability of this function aswell!